cmd/collectproto/README.md

```diff
@@ -0,0 +1,17 @@
+# `collectproto`
+
+Collect, combine and produce a single protobuf file with all declarations.
+
+This program reads provided protobuf declarations and:
```

Comment:
At first I thought you found this program. Then I realized you wrote it.
Above and beyond the call of duty!

Reply:
At first I only typed `package main`. The rest _just_ happened :wink:

Reply:
No regexp? How dare you...